docker/vm/sestatus

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+echo "Current mode:                   permissive"
```

Comment:
Does this actually disable SELinux or just the reporting functions?

Reply:
Since SELinux isn't really running in Docker containers, this just tricks Ansible into passing its "is selinux set up correctly?" test.